lib/views/home/skeleton_loading/skeleton_placeholder.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flutter/material.dart';
+
+enum TextPlaceholderType {
+  oneLine,
+  twoLines,
+}
+
+class TextPlaceholder extends StatelessWidget {
+  final TextPlaceholderType type;
+  final double width;
+
+  const TextPlaceholder({
+    Key? key,
+    required this.type,
+    required this.width,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 16.0),
+      child: _buildLines(),
+    );
+  }
+
+  Widget _buildLines() {
```

Comment:
@nmint8m Optional, we can optimize the code like below
```suggestion
  Widget _buildLines() {
   return Column(
     children: [
        Container(
            width: width,
            height: 12.0,
            color: Colors.white,
          ),
          if(type == TextPlaceholderType.twoLines)
          ..[
             const SizedBox(height: 8.0),
             Container(
                width: width * 0.6,
                height: 12.0,
                color: Colors.white,
          ),
          ]
     ]
   );
  }
```